adv-java/src/main/java/dataModel/Delete.java

```diff
@@ -0,0 +1,27 @@
+package dataModel;
+import sqlDao.DeleteServlet; 
+import java.io.IOException;
+import javax.servlet.http.*;
+
+public class Delete extends HttpServlet{
+
+	public void service(HttpServletRequest req,HttpServletResponse res) throws IOException{
+	
+		HttpSession session=req.getSession();
+		int Pid =(int) session.getAttribute("pid");
+		String mailId = (String) session.getAttribute("mailid");
+	
+		System.out.println("1"+Pid+mailId);
```

Comment:
If these were used during debugging, these should be removed from final the code base. Also, I see this class is not properly formatted.